protocol/x/delaymsg/keeper/dispatch_test.go

```diff
@@ -32,12 +32,12 @@ var (
 
 	DelayMsgAuthority = authtypes.NewModuleAddress(types.ModuleName).String()
 
-	BridgeGenesisAccountBalance = sdk.NewCoin("dv4tnt", sdkmath.NewInt(1000000000))
-	AliceInitialAccountBalance  = sdk.NewCoin("dv4tnt", sdkmath.NewInt(99500000000))
+	testDenom = "dv4tnt"
+
+	BridgeGenesisAccountBalance = sdk.NewCoin(testDenom, sdkmath.NewInt(1000000000))
 
 	delta                        = constants.BridgeEvent_Id0_Height0.Coin.Amount.Int64()
-	BridgeExpectedAccountBalance = sdk.NewCoin("dv4tnt", sdkmath.NewInt(1000000000-delta))
-	AliceExpectedAccountBalance  = sdk.NewCoin("dv4tnt", sdkmath.NewInt(99500000000+delta))
+	BridgeExpectedAccountBalance = sdk.NewCoin(testDenom, sdkmath.NewInt(1000000000-delta))
```

Comment:
I don't think this and the above line should need to use casing to `int64` (?) why not just do math using `sdkmath.Int` directly?

Also should this line use `BridgeGenesisAccountBalance` or does it happen to use the same number for some unrelated reason

Reply:
Yup, refactored to using `sdkmath.Int` math on `BridgeGenesisAccountBalance` and `delta`